MatCalciteTargetPlatform/MatCalciteTargetPlatform.target

```diff
@@ -9,5 +8,21 @@
 <unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
 <repository location="http://download.eclipse.org/releases/photon/"/>
 </location>
+	<location includeDependencyDepth="infinite" includeDependencyScopes="compile,provided,runtime" includeSource="true" missingManifest="generate" type="Maven">
+		<dependencies>
+			<dependency>
+				<groupId>org.apache.calcite</groupId>
+				<artifactId>calcite-core</artifactId>
+				<version>1.36.0</version>
+				<type>jar</type>
+			</dependency>
+		</dependencies>
+	</location>
```

Comment:
Thank you, however, I think I have made the very same change in https://github.com/vlsi/mat-calcite-plugin/pull/31/files#diff-16b1c688a2d84e7f3c88da892a116779943cc252b1157148f597638612f6035cR24-R28, and it does not make the dependency to be available during the build time.

Reply:
This is similar (but not the same) `wrapAsBundle` is only usefull if you don't have (or want) a target, also please carefully read the build output as **the bundle name is most likely not what you might expect it to be**. Also you should *not* list all bundles you should only import the packages required in your code, the rest will be resolved automatically!